src/iperf_util.c

```diff
@@ -432,3 +432,57 @@ iperf_dump_fdset(FILE *fp, char *str, int nfds, fd_set *fds)
     }
     fprintf(fp, "]\n");
 }
+
+/* daemon(3) implementation for systems lacking one */
+#ifndef HAVE_DAEMON
+int daemon(int nochdir, int noclose)
+{
+    pid_t pid = 0;
```

Comment:
I'd rewrite this implementation to do the order of operations we previously described, use `dup2()`, etc.

Reply:
Yeah I just didn't want to deal with all those changes at the same time.  :-)

It's also interesting that what's in the patch very roughly corresponds to the most readily-available source for daemon(3) that I have (FreeBSD).  The order of operations you gave, is that based on glibc or something else?  I was going to dig through various sources to see what to do...still trying to wrap my mind around the idea that there are multiple ways to do daemon(3) (and yeah I know it hasn't really been standardized so that's probably to be expected).